packages/vaadin-virtual-list/test/unlimited-size.test.js

```diff
@@ -0,0 +1,174 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync, oneEvent } from '@vaadin/testing-helpers';
+import { Virtualizer } from '../virtualizer';
+
+describe('unlimited size', () => {
+  let virtualizer;
+  let scrollTarget;
+  let elementsContainer;
+
+  function init(config = {}) {
+    scrollTarget = fixtureSync(`
+      <div style="height: 100px;">
+        <div></div>
+      </div>
+    `);
+    const scrollContainer = scrollTarget.firstElementChild;
+    elementsContainer = scrollContainer;
+
+    virtualizer = new Virtualizer({
+      createElements: (count) => Array.from(Array(count)).map(() => document.createElement('div')),
+      updateElement: (el, index) => {
+        el.index = index;
+        el.id = `item-${index}`;
+        el.textContent = el.id;
+      },
+      scrollTarget,
+      scrollContainer,
+      ...config
+    });
+
+    virtualizer.size = 1000000;
+  }
+
+  beforeEach(() => init());
```

Comment:
We probably can merge `init` call into the `beforeEach` here, there are no other calls in this file.

Reply:
Done